followthemoney/types/common.py

```diff
@@ -57,7 +57,12 @@ class PropertyType(object):
     other entities that mention the same phone number, email address or name as the
     one currently seen by the user."""
 
-    max_size: Optional[int] = None
+    max_size: int = 250
+    """The maximum length of a single value of this type. This is used to avoid
+    the individual values that may be malformed or too long to be stored in
+    downstream databases with fixed column lengths."""
```

Comment:
Does "length" refer to number of bytes, codepoints, displayed characters, or is it intentionally ambiguous? Would be good to clarify this.

Reply:
Fair point! I would angle for codepoints, do you have a preference?

Reply:
Postgres calls it characters, perhaps a good reference? https://www.postgresql.org/docs/current/datatype-character.html

Reply:
Don't really have a strong preference. I think bytes is probably the most practical as databases with fixed column lengths tend to be in bytes (I think? Not entirely sure) and most RFCs and such tend to specify maximum sizes in bytes. I'm also okay with codepoints.

Reply:
Didn't see your second comment – looks like MariaDB also uses "characters" rather than "bytes", so looks like I remembered wrong.

Reply:
Added to the comment.